src/main/java/org/kitteh/irc/client/library/Client.java

```diff
@@ -829,4 +832,14 @@ default void sendMultiLineNotice(@Nonnull MessageReceiver target, @Nonnull Strin
      * @param reason quit message to send
      */
     void shutdown(@Nonnull String reason);
+
+    /**
+     * Sends a DCC CHAT request to the target.
+     *
+     * <p>When the server socket is bound locally, a
+     * {@link DCCSocketBoundEvent} will be fired. When the chat is connected,
+     * a {@link DCCConnectedEvent} will be fired. If the connection fails, a
+     * {@link DCCFailedEvent} will be fired.</p>
+     */
+    void requestDCCChat(@Nonnull User target);
```

Comment:
So there's a way to request a new DCC chat, but I see no way to get existing ones.